tests/workout.test.js

```diff
@@ -27,39 +34,124 @@ describe('Workout Model', () => {
     time_per_exercise: 60
   }
 
-  describe('Workout validations: ', () => {
+  describe('Model: ', () => {
+
+    describe('Workout validations: ', () => {
+
+      it('successfully creates a valid workout', () => {
+        return Workout.create(validWorkout)
+          .then(createdWorkout => {
+            expect(createdWorkout).to.contain(validWorkout)
+          })
+          .catch(err => console.error(err))
+      })
+
+      it('reports a validation error for invalid workout entries', () => {
+        return Workout.create(invalidWorkout)
+          .then(error => {
+            expect(error).to.be.instanceOf(Error)
+            expect(error.message).to.contain('invalid input')
+          })
+          .catch(err => console.log(chalk.green('You got a validation error')))
+      })
 
-    it('successfully creates a valid workout', () => {
-      return Workout.create(validWorkout)
-        .then(createdWorkout => {
-          expect(createdWorkout).to.contain(validWorkout)
-        })
-        .catch(err => console.error(err))
     })
 
-    it('reports a validation error for invalid workout entries', () => {
-      return Workout.create(invalidWorkout)
-        .then(error => {
-          expect(error).to.be.instanceOf(Error)
-          expect(error.message).to.contain('invalid input')
-        })
-        .catch(err => console.log(chalk.green('You got a validation error')))
+    describe('Has Correct Associations: ', () => {
+
+      it('creates associations for patient_id, plan_id, and treatment_id', () => {
+        return Workout.create(validWorkout)
+          .then(workout => {
+            expect(workout.patient_id).to.equal(null)
+            expect(workout.plan_id).to.equal(null)
+            expect(workout.treatment_id).to.equal(null)
+          })
+          .catch(err => console.error(err))
+      })
+    })
+  })
+
+  describe('Routes: ', () => {
+
+  before('create a fake patient', () => {
+    return Patient.create({
+      id: 1001,
+      first_name: 'Fake',
+      last_name: 'Guy',
+      email: 'fakeguy@gmail.com',
+      DOB: '01/01/1980',
+      gender: 'M'
+    })
+    .then(ok => console.log(chalk.green('fake patient created')))
+    .catch(err => console.error(err))
+  })
+
+  before('ensures a fake plan exists', () => {
+    return Plan.create({
+      id: 1001,
+      duration: 24,
+      therapy_focus: 'Post Tommy John\'s surgery',
+      notes: 'Even famous MLB physical therapists use Steps',
+      patient_id: 1001
     })
+    .then(ok => console.log(chalk.green('fake plan created')))
+    .catch(err => console.error(err))
+  })
 
+  after('delete that fake patient', () => {
+    return Patient.findById(1001)
```

Comment:
Would you delete the plan instead of the patient? I would expect we would delete the plan but keep the patient in the database so can have patient in database without plan